src/tike/ptycho/ptycho.py

```diff
@@ -494,8 +504,10 @@ def iterate(self, num_iter: int) -> None:
                     self.parameters.probe,
                     self.parameters.object_options.preconditioner,
                 )))
-
-            if self.parameters.eigen_probe is not None:
+            # ELSE IF: self.parameters.probe_options.rescale_using_fixed_probe_intensity_photons
+            # ELSE:    do nothing? or should the "rescale_using_mean_of_abs_object" be default?
```

Comment:
If you want to provide multiple options that are exclusive, then probably, we want a `rescale_method` parameter instead of a boolean for each method.

Reply:
yes, I was going to do something like this for the next pull request on controlling probe scaling...we'll revisit this part

Reply:
Instead of introducing the new parameter "rescale_using_mean_of_abs_object_period" then removing it later. Just skip to implementing "rescale_method", but only offer one option. That way, there are fewer API breaking changes for Steve to adapt to.